doc/models_library/index.rst

```diff
@@ -1,386 +1,49 @@
 Models library
 ==============
 
-Neuron models
```

Comment:
This file still needs to be hand-tuned, unfortunately. Could you leave it as-is for this PR? We will pick it up again in #920.